index.js

```diff
@@ -15,100 +18,112 @@ function hasImplicitLicense(commit_message) {
 
   // Skip the commit subject (first line)
   for (var i = 1; i < lines.length; i++) {
-      // Remove any trailing `\r` char
-      const line = lines[i].replace(/\r$/,'');
-      const license = line.match(/^License: ?(.+)$/);
-      if (license && implicitLicenses.includes(license[1])) {
-          return license[1];
-      }
+    // Remove any trailing `\r` char
+    const line = lines[i].replace(/\r$/, '');
+    const license = line.match(/^License: ?(.+)$/);
+    if (license && implicitLicenses.includes(license[1])) {
+      return license[1];
+    }
   }
   return '';
 }
 
 async function run() {
   // Install dependencies
-  core.startGroup('Installing python3-launchpadlib')
+  core.startGroup('Installing python3-launchpadlib');
   await exec.exec('sudo apt-get update');
   await exec.exec('sudo apt-get install python3-launchpadlib');
-  core.endGroup()
+  core.endGroup();
```

Comment:
please move all the styling changes into a separate PR

Reply:
@beliaev-maksim done! plz let me know if anything still looks opinionated